agir/notifications/components/NotificationSettings/NotificationSettingPanel.js

```diff
@@ -197,6 +198,11 @@ const NotificationSettingPanel = (props) => {
         Vous recevez les e-mails sur votre adresse <u>{user.email}</u>&nbsp;
         <Link route="personalInformation">(modifier)</Link>
       </p>
+      <div style={{ marginLeft: "20px", marginBottom: "20px" }}>
+        <Button small link href={nonReactRoutes.contactConfiguration}>
```

Comment:
Ça suffit comme ça normalement : 
```suggestion
        <Button small link route="contactConfiguration">
```

Reply:
C'est pourtant plus générique d'utiliser la route depuis le fichier `routes` non ?

Reply:
Ca me parait moins générique que d'utiliser la variable mais ok :)